### PR TITLE
Add time command to modify raid end time

### PR DESCRIPTION
**Pull request by nicolasblaye**
I did no unit test as it is in the main class. This should move to a cog later and be tested

### CODE REVIEW

**Comment by nicolasblaye** [comment]
should resolve #21

File: mod_bot/main.py [hunk 1]
```diff
@@ -213,6 +213,24 @@ def add_minutes(hstart, mstart, mn):
...
+        for fmt in ('%Hh%M', '%H:%M'):
+            try:
+                if new_time is None:
+                    new_time = datetime.datetime.strptime(words[1], fmt).time()
+            except ValueError:
+                pass
+        if new_time is not None:
+            new_name = '-'.join(cwords[:-1] + [new_time.strftime('fin%Hh%M')])
+            await message.channel.edit(name=new_name)
+            await message.channel.send('@here Heure de fin de raid mise à jour : ' + new_time.strftime('%Hh%M'))
```
Comment (tama) [inline mod_bot/main.py]: Not sure about the @here in the message, should we really mention every one online because of the change ?

Comment (nicolasblaye) [reply mod_bot/main.py]: true, however do you think we should at least mention the participants?

Comment (tama) [reply mod_bot/main.py]: @nicolasblaye sorry for the late reply ; mentionning the participants seems to be a good compromise, but "outsiders" may not see the change ; let's leave the @here first and we'll edit it later if users are not happy (or even better, some kind of "opt-in" where administrators can choose if it mentions people or not)

**Pull request merged**